static/js/shared/place_search_bar.tsx

```diff
@@ -143,7 +143,7 @@ class PlaceSearchBar extends Component<PlaceSearchBarPropType> {
             </span>
           </div>
         );
-        container.prepend(ReactDOM.render(result, container));
+        ReactDOM.render(result, container);
```

Comment:
curious if removing container.prepend changes anything on the UI?

Reply:
No, it was a misuse, ReactDOM.render does not return anything